onnxconverter_common/onnx2py.py

```diff
@@ -65,12 +65,20 @@ def make_node(op_type, inputs, outputs, name=None, doc_string=None, domain=None,
     order_repeated_field(node.attribute, 'name', kwargs.keys())
     return node
 
+
+def make_graph(*args, doc_string=None, **kwargs):
+    graph = helper.make_graph(*args, doc_string=doc_string, **kwargs)
```

Comment:
it looks a bug that should be fixed in ONNX repo.

Reply:
Arguably empty doc strings don't need to be included, but I'm trying to make the results byte-for-byte identical.  I would appreciate it if the onnx helpers let me do this.